### DISCUSSION TITLE
Cannot download a YouTube video

### DISCUSSION THREAD

**Discussion by elusis** [aclap-dev/video-downloadhelper#175] [Help and Q&A]
### Your question

Hi there,

I am using the latest version with Firefox 122.0.1 for Mac. (I normally use Chrome but I know YouTube cannot be downloaded via Chrome.)  I am trying to download [this video](https://www.youtube.com/watch?v=7aE7dlEqar0) which has vanished from Vimeo and other sites where I used to use it with a class for sex therapists - can't even find it on DVD! VDH says "No media to process in the current tab" even once I hit play on the video.  I would welcome any help

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 6bf8afc90bb445b1
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/dr. elusis",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by GrampaWildWilly** [comment]
I had this whole long explanation of how to get this content I was going to post.  Sadly, the content for this video is not like anything I've dealt with before.  I was able to download a video behind a log in yesterday.  But this one isn't showing me the same sort of content.  I am greatly disappointed.  This one is going to have to be left for better minds than mine.

**Comment by Nivminko** [comment]
I also cannot download videos from YouTube and i have the premium app

**Reply by GrampaWildWilly** [reply]
You need to click the Wiki link at the top of this page.  That is a user manual of sorts.  Down the right side of the page there is a Table of Contents.  Each chapter is a clickable link.  Click the one for "I need help."  Read that & follow the instructions.  We need a proper problem report in order to investigate your issue.  The original poster in this thread read those instructions & followed them.  You need to post something like that.  I have to say that in general, I have no problem downloading content from YouTube.  The particular video mentioned by the original poster happened to resist downloading by VDH.  But most other content I encounter on YouTube downloads easily with VDH.  I am running Windows 11 Pro 64-bit version 23H2 build 22631.3296, Firefox 123.0.1 64-bit, licensed VDH 8.2.2.7a1 beta, CoApp 2.0.19.  This configuration works just fine with most YouTube content.  I don't work for VDH.  I'm just another user.
